{"name": "Download aquivo", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <div class=\"fundo-documentos-tipo\" id=\"Demonstração Financeira\">...</div> is not clickable at point (391, 544). Other element would receive the click: <div class=\"sc-gzVnrw goUxVs\">...</div>\n  (Session info: headless chrome=90.0.4430.212)\n\n", "trace": "  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\navegacao_step.py\", line 53, in step_impl\n    context.vortx.clicar_download(context)\n  File \"C:\\Users\\luis.gimenez\\Documents\\TesteVortx\\features\\pages\\NavegacaoPage.py\", line 101, in clicar_download\n    self.waitShort.until(EC.presence_of_element_located((By.ID, \"Demonstração Financeira\"))).click()\n  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Dado que esteja home page da Vortx", "status": "passed", "start": 1621228534905, "stop": 1621228535107}, {"name": "Dado que eu esteja no detalhe de um investimento", "status": "passed", "start": 1621228535108, "stop": 1621228535335}, {"name": "Quando eu clicar na aba documentos", "status": "passed", "start": 1621228535335, "stop": 1621228535955}, {"name": "E clicar em download do pdf", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <div class=\"fundo-documentos-tipo\" id=\"Demonstração Financeira\">...</div> is not clickable at point (391, 544). Other element would receive the click: <div class=\"sc-gzVnrw goUxVs\">...</div>\n  (Session info: headless chrome=90.0.4430.212)\n\n", "trace": "  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\navegacao_step.py\", line 53, in step_impl\n    context.vortx.clicar_download(context)\n  File \"C:\\Users\\luis.gimenez\\Documents\\TesteVortx\\features\\pages\\NavegacaoPage.py\", line 101, in clicar_download\n    self.waitShort.until(EC.presence_of_element_located((By.ID, \"Demonstração Financeira\"))).click()\n  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\luis.gimenez\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1621228535956, "stop": 1621228537273}, {"name": "Então o sistema deve realizar download", "status": "skipped", "start": 1621228537276, "stop": 1621228537276}], "start": 1621228534904, "stop": 1621228537276, "uuid": "0c243e7e-1da5-4ff7-b806-754c8ecff163", "historyId": "828bafe26cdb17d2c7a50e0879d71482", "fullName": "Desafio - Laboratório Automação: Download aquivo", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "cenario3"}, {"name": "feature", "value": "Desafio - Laboratório Automação"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}